### ISSUE TITLE
Error in hl.split_multi_hts documentation

### ISSUE THREAD

**Issue by jsture** [hail-is/hail#10676]
Hail version: 0.2.71-f3a54b530979

In hl.split_multi_hts under "Warning" it says:

```
>>> bi = mt.filter_rows(hl.len(mt.alleles) == 2)
>>> bi = bi.annotate_rows(was_split=False)
>>> multi = mt.filter_rows(hl.len(mt.alleles) > 2)
>>> split = hl.split_multi_hts(multi)
>>> mt = split.union_rows(bi)
```

This throws:

```

ValueError: 'MatrixTable.union_rows' expects row types for all datasets to be the same. Found:    
dataset 0: struct{locus: locus<GRCh38>, alleles: array<str>, rsid: str, qual: float64, filters: set<str>, info: struct{AF: array<float64>, AQ: array<int32>, AC: array<int32>, AN: int32}, a_index: int32, was_split: bool}    
dataset 1: struct{locus: locus<GRCh38>, alleles: array<str>, rsid: str, qual: float64, filters: set<str>, info: struct{AF: array<float64>, AQ: array<int32>, AC: array<int32>, AN: int32}, was_split: bool, a_index: int32}
```
Maybe it should instead be?

```
>>> bi = mt.filter_rows(hl.len(mt.alleles) == 2)
>>> bi = bi.annotate_rows(a_index=1, was_split=False)
>>> multi = mt.filter_rows(hl.len(mt.alleles) > 2)
>>> split = hl.split_multi_hts(multi)
>>> mt = split.union_rows(bi)
```

**Comment by johnc1231** [comment]
Thanks for raising this, I'll check it out next week.

**Comment by johnc1231** [comment]
Fixed by #10781, website will update when we release hail 0.2.75
